### PR TITLE
feat: add pl-PL

### PR DESCRIPTION
**Pull request by tipakA**
Adds polish localization. Still needs to resolve some quirks though, that's why draft.

### CODE REVIEW

File: src/i18n/lang/pl-PL.lang.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	public override emote_copy_unknown_download_error = 'Podczas pobierania emoji wystąpił nieznany błąd.';
+	public override emote_copy_unknown_upload_error = 'Podczas przesyłania emoji na serwer wystąpił nieznany błąd.';
+	public override emote_copy_success = (name: string, newEmoji: string): string =>
+		`:tada:  Skopiowano \`:${name}:\` na Twój serwer! ${newEmoji}`;
+	public override emote_copy_no_emotes = 'Musisz podać przyynajmniej 1 emoji do skopiowania.';
+	public override emote_copy_multiple_success = (failed: number, success: number): string =>
+		`Pomyślnie przesłano ${success} emoji! (Niepomyślne: ${failed})`;
+
+	// TODO: ????
+	// TODO: Uh, you do know that managing emoji is done in discord anyway, and that since 8.45g from march 2019 android has ways to manage emotes as well, right?
```
Comment (memikri) [inline src/i18n/lang/pl-PL.lang.ts]: Yes, I'm aware mobile has this capability; however, there were numerous people in the support server who asked me to "add support for mobile," so I added mobile support to the description.

File: src/i18n/lang/pl-PL.lang.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	public override info_remoji_server_invite = (invite: string): string => `**[DOŁĄCZ DO SERWERA REMOJI](${invite})**`;
+	public override info_remoji_bot_field = 'Dodaj Remoji!';
+	// TODO: Zaproszenie sounds bad, maybe address or like "click (here)"
+	public override info_remoji_bot_invite = (appId: string, permissionBits: string): string =>
+		`**[ADRES](https://discord.com/oauth2/authorize?client_id=${appId}&permissions=${permissionBits}&scope=applications.commands%20bot)**`;
+	public override info_remoji_vote_field = 'Głosuj na Remoji!';
+	public override info_remoji_vote_value = (url: string): string => `**[GŁOSUJ NA TOP.GG](${url})**`;
+	// TODO: e? y? a?
+	public override info_remoji_created = 'Stworzone przez Shino.';
+	// TODO: comma?
```
Comment (memikri) [inline src/i18n/lang/pl-PL.lang.ts]: Comma where?

File: src/i18n/lang/pl-PL.lang.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	public override image_unknown_error = 'Nie udało się pobrać obrazu z powodu nieznanego błędu.';
+	public override image_upload_success = (name: string, emoji: string): string =>
+		`:tada:  Przesłano \`:${name}:\` na Twój serwer! ${emoji}`;
+	public override image_upload_failed_unknown_error =
+		'Nie udało się przesłać emoji z powodu nieznanego błędu. Spróbuj użyć innego obrazu lub wysłać obraz jako załącznik na Discordzie.';
+	public override language_change_success = (oldLocale: Nullable<I18NLanguage>, newLocale: I18NLanguage): string =>
+		`Pomyślnie zmieniono język${oldLocale ? ` z \`${oldLocale}` : ''} na \`${newLocale}\``;
+
+	public override embed_help_callout_title = 'Potrzebujesz pomocy?';
+	// TODO: branding?
```
Comment (memikri) [inline src/i18n/lang/pl-PL.lang.ts]: ?

**Comment by memikri** [comment]
@tipakA still a draft?

**Comment by tipakA** [comment]
Should be good to go now.

**Pull request merged**